src/controllers/redirect.controller.ts

```diff
@@ -43,15 +50,21 @@ export class RedirectController extends BaseController {
       return this.redirect(ENDORSE_COMPANY_CLOSURE_CERTIFICATE_URI)
     }
 
+    if (dissolution!.application_status === ApplicationStatus.PENDING_APPROVAL && this.getUserAlreadySigned(dissolution, userEmail)) {
+      return this.redirect(WAIT_FOR_OTHERS_TO_SIGN_URI)
+    }
```

Comment:
According to https://github.com/companieshouse/dissolution-web#controller-flow we should only navigate to the "Wait for others to sign confirmation page" if the user is the applicant.

We can identify if the user is an applicant by:
```
const isApplicant: boolean = dissolution.created_by === userEmail
```